pkg/storage/replica_gc_queue.go

```diff
@@ -183,6 +203,7 @@ func (q *replicaGCQueue) process(
 	if _, currentMember := replyDesc.GetReplicaDescriptor(repl.store.StoreID()); !currentMember {
 		// We are no longer a member of this range; clean up our local data.
 		log.VEventf(ctx, 1, "destroying local data")
+		rgcq.metrics.RemoveReplicaCount.Inc(1)
```

Comment:
note that there is another RemoveReplica call lower down, but that's the (ostensibly) unused merge path. I could add a separate metric for that, if you think it's worth it (it will probably always be zero)